app/scripts/polling_location_finder.js

```diff
@@ -78,7 +78,9 @@ define(['jquery', 'geojson', 'json!vendor/ELECTIONS_WardsPrecincts.geojson', 'js
         return encodeURI(url + destination);
     }
 
-    return function(latLng) {
+    // successCallback & errorCallback are optional parameters,
+    // which will be executed on success or failure of the google maps direction service, respectively
```

Comment:
I think you can do without the comments- I think "callback" in the variable name is self-documenting, and the `if` statements make it clear that they're optional (and this is a common enough pattern).